### PR TITLE
Add Rodolfo Martinez to Spanish approver team

### PR DESCRIPTION
**Pull request by electrocucaracha**
### Describe your changes

Given his active [collaboration](https://github.com/cncf/glossary/pulls?q=author%3Aramrodo) in Spanish localization team, this PR recognizes the effort made by @ramrodo and promotes him to the approver role.

/cc @ramrodo @krol3

### Related issue number or link (ex: `resolves #issue-number`)
NA

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by ramrodo** [comment]
Thanks! Happy to continue contributing :)

**Review by krol3 — APPROVED** [review]
LGTM

**Review by raelga — APPROVED** [review]

**Comment by seokho-son** [comment]
Maintainers are discussing extending the max number of approvers(reviewers) for each localization team from 4 to 5.
So that, folks who have leadership in the localization team can keep stay with the leadership.
Anyway, please wait for a while for the outcome.

**Comment by seokho-son** [comment]
Hi @electrocucaracha 
In the latest Glossary meeting, we agreed to accept additional approvers for an ongoing localization team. I think we can remain the existing approvers for Spanish team and add @ramrodo .

File: .github/settings.yml [hunk 1]
```diff
@@ -125,7 +125,7 @@ collaborators:
     permission: push
     
   # l10n es approvers
-  - username: raelga
+  - username: ramrodo
```
Comment (raelga) [inline .github/settings.yml:128]:
```suggestion
  - username: raelga
    permission: push

  - username: ramrodo
```

File: .github/settings.yml [hunk 2]
```diff
@@ -334,7 +334,7 @@ branches:
         apps: []
         # es approvers
         users:
-         - raelga
+         - ramrodo
```
Comment (raelga) [inline .github/settings.yml:337]:
```suggestion
         - raelga
         - ramrodo
```

File: CODEOWNERS [hunk 3]
```diff
@@ -34,7 +34,7 @@
 /content/bn/ @mitul3737 @Mouly22 @ikramulkayes @Imtiaz1234
 
 # Approvers for Spanish contents
-/content/es/ @raelga @electrocucaracha @krol3 @92nqb
+/content/es/ @ramrodo @electrocucaracha @krol3 @92nqb
```
Comment (raelga) [inline CODEOWNERS:37]:
```suggestion
/content/es/ @raelga @ramrodo @electrocucaracha @krol3 @92nqb
```

**Review by raelga — CHANGES_REQUESTED** [review]
Adds @ramrodo along current approvers

**Review by raelga — APPROVED** [review]

**Review by seokho-son — APPROVED** [review]
LGTM!

Hi @ramrodo please check and confirm following policies :)
[Policies for localization approvers (to be confirmed by candidates) #723](https://github.com/cncf/glossary/discussions/723)

**Comment by ramrodo** [comment]
> Hi @ramrodo welcome to the Spanish approver team! please check and confirm following policies :) [Policies for localization approvers (to be confirmed by candidates) #723](https://github.com/cncf/glossary/discussions/723)

Thanks, @seokho-son. I'll do.

**Pull request merged**